app/Main.hs

```diff
@@ -32,6 +33,12 @@ contracts = daoContractRegistry
     , dciConfig = GameDAO.config
     , dciExtraParser = pure def
     }
+  -- TODO: Update CI to upload RegistryDAO documentation
```

Comment:
Oh, that's not what I meant :slightly_smiling_face: 

I'd like you to update CI to upload RegistryDAO _now_ and also add a comment here saying that one should update CI config to upload new documentation when they add a new contract to this list. For example, when we add TreasuryDAO, we want to upload its documentation.

Reply:
Got this, I've added this to the CI now.